src/components/pages/group/form/uploader/imageCard.tsx

```diff
@@ -0,0 +1,88 @@
+import { CloseButton, Flex, Image } from "@chakra-ui/core";
+import styled from "@emotion/styled";
+import { AssetStatus, IDBObservationAsset } from "@interfaces/custom";
+import { ASSET_TYPES, LOCAL_ASSET_PREFIX } from "@static/observation-create";
+import {
+  getFallbackByMIME,
+  getMyUploadsThumbnail,
+  getObservationThumbnail,
+  getYoutubeImage
+} from "@utils/media";
+import { useStoreState } from "easy-peasy";
+import React, { useMemo } from "react";
+
+import StatusIcon from "@components/pages/observation/create/form/uploader/statusicon";
+import useObservationCreate from "@components/pages/observation/create/form/uploader/use-observation-resources";
```

Comment:
Please sort imports properly
use something like this for better DX https://marketplace.visualstudio.com/items?itemName=rbbit.typescript-hero